### PR TITLE
Canonicalize CRAN links

### PR DESCRIPTION
**Pull request by katrinleinweber**
- [~] closes #xxxx
- [~] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [~] whatsnew entry

### CODE REVIEW

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -1026,7 +1026,7 @@ unit interval). The point in the plane, where our sample settles to (where the
 forces acting on our sample are at an equilibrium) is where a dot representing
 our sample will be drawn. Depending on which class that sample belongs it will
 be colored differently.
-See the R package `Radviz <https://cran.r-project.org/web/packages/Radviz/>`__
+See the R package `Radviz <https://cran.r-project.org/package=Radviz/>`__
```
Comment (WillAyd) [inline doc/source/user_guide/visualization.rst]: So when pasting this in my browser it redirects to the URL prior to this commit - is there any reason for changing this?

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: doc/source/user_guide/visualization.rst (hunk 1 above)
Comment (jschendel) [reply doc/source/user_guide/visualization.rst]: At the bottom of the page it says:

>Please use the canonical form https://CRAN.R-project.org/package=Radviz to link to this page.

It's a little strange that it redirects back to the original URL but seems to be CRAN's preference to use those links.

**Review by jschendel — APPROVED** [review]

File: doc/source/user_guide/visualization.rst (hunk 1 above)
Comment (WillAyd) [reply doc/source/user_guide/visualization.rst]: Hmm that's weird. Makes sense for this PR then!

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @katrinleinweber !

**Comment by katrinleinweber** [comment]
You're welcome, and thanks for merging :-) Sorry about the confusion! I forgot to paste the CRAN docu URL into the PR description: https://cran.r-project.org/doc/manuals/R-exts.html#Specifying-URLs

**Pull request merged**